index.html

```diff
@@ -631,13 +644,32 @@ <h1>Terminology</h1>
       </dt>
       <dd>The combination of Public Security Metadata, Private Security Data, and any other configuration
         information (such as public keys) necessary to operationally configure the security mechanisms of a Thing.</dd>
+      <dt>
+        <dfn>Service</dfn>
```

Comment:
Should we be using the term WebService?

Reply:
In particular, if it is a software component without a network interface, it is usually called a "module" or a "library".  When I hear the word "service" I immediately assume it has a network interface.